### PR TITLE
📚  update T1K docs

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

Updating the T1k docs to reflect the current state of the tool and new recommendations.

Part of https://github.com/d3b-center/bixu-tracker/issues/2332

## Type of change

Please delete options that are not relevant.

- [x] documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Just docs

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have committed any related changes to the PR

### CODE REVIEW

File: docs/T1K_README.md [hunk 1]
```diff
@@ -1,27 +1,47 @@
...
+
+ - `reads`: Port for `-1` input, read1 fastq reads if paired-end (omit if single end or interleaved)
+ - `mates`: Port for `-2` input, mates (read2) fastq reads if paired-end (omit if single end or interleaved)
+ - `single_end`: Port for `-u` input, fastq reads if single-end (omit if paired-end or interleaved)
+ - `interleaved`: Port for `-i` input, fastq reads if interleaved (omit if single end, or read1 and mates are separate files)
+ - `bam`: Port for `-b` input, if bam input (use if input reads are bam and not fastq)
+
+### Presets
+
+T1K can be used for RNA-seq, WES, or WGS analysis. The default parameters are for RNA-seq. If you wish to perform a different analysis, you can change the `preset` field. That field has the following options:
```
Comment (migbro) [inline docs/T1K_README.md:23]: You say the defaults are for RNAseq, but I am guessing it's also for WXS given there is a specific WGS preset

File: docs/T1K_README.md [hunk 2]
```diff
@@ -1,27 +1,47 @@
...
+## Inputs
+### Reference
  - `t1k_ref_tar`: Tar ball with T1K reference
  - `t1k_ref_path`: String of path to un-tarred hla ref, i.e. kiridx/kiridx_rna_seq.fa
  - `output_basename`: Prefix string for output file names. Default inferred from input
-### Optional resource
+
+ ### Input Reads
+
+The reads for T1K can be delivered as either BAM or fastq. The [T1K documentation](https://github.com/mourisl/T1K/tree/v1.0.5#inputoutput) outlines the various ways in which these files can be delivered. These match this tool's inputs:
```
Comment (migbro) [inline docs/T1K_README.md:13]: ✨Now with CRAM support!✨

**Review by migbro — CHANGES_REQUESTED** [review]
Two minor suggestions

**Comment by migbro** [comment]
@dmiller15 found a couple more requests for added info here: https://github.com/d3b-center/bixu-tracker/issues/2332#issuecomment-2032994819

**Review by migbro — APPROVED** [review]
Looks good, thanks!

**Review by wongjessica93 — APPROVED** [review]

**Pull request merged**